nixos/modules/services/monitoring/adguard-exporter.nix

```diff
@@ -0,0 +1,117 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.adguard-exporter;
+in {
+  options.services.adguard-exporter = {
```

Comment:
Hey, don't we generally

![image](https://github.com/NixOS/nixpkgs/assets/42209822/a8f723c5-eb3b-41f4-b49d-0c230f2cd250)